app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -531,6 +531,10 @@ private void handleNavIntent() {
             bottomSheetCallback.onSlide(null, 1.0f);
         } else if (Intent.ACTION_VIEW.equals(intent.getAction())) {
             handleDeeplink(intent.getData());
+        } else if (Intent.ACTION_CREATE_SHORTCUT.equals(intent.getAction())) {
+            intent = new Intent(this, SelectSubscriptionActivity.class);
+            intent.setAction(Intent.ACTION_CREATE_SHORTCUT);
+            startActivity(intent);
```

Comment:
Can't you just add `SelectSubscriptionActivity` directly to the manifest instead of opening `MainActivity` that then starts it?